### DISCUSSION TITLE
Tsl2561 , divide value??

### DISCUSSION THREAD

**Discussion by lisergio** [arendst/Tasmota#21935] [Q&A] [answered]
Hello!
I have installed a wind sensor and a tsl2561 light sensor to control two curtains with homekit,
I have adjusted the analog input for the wind sensor and pins 4 and 5 as sda and scl for the tsl2561.
I have created the accessories with homebridge, and everything works correctly.
But the homekit automations only allow adjustment up to 6000lux and when I want the awning to go down, it shows 20000lux...
My question is, can I make any adjustment in tasmota to divide the value by 4 or 5???
Thanks!!

**Comment by joba-1** [comment]
either software hack:

https://github.com/arendst/Tasmota/blob/663fc88373ca8e4ca9cc222297e2e337acaef659/tasmota/tasmota_xsns_sensor/xsns_16_tsl2561.ino#L60

insert division before this line.

or hardware hack:

put sunglasses in front of the sensor 

or tinker

use berry (esp32 only) or rules to publish calculated values (not my area of expertise)

**Reply by sfromis** [reply, marked as answer]
Layers of paper until sufficient attenuation may be the simplest hack.

**Comment by Noschvie** [comment]
> tsl2561 light sensor

I2C-Adresse
ADR SEL LOW : 0x29
ADR SEL FLOAT: 0x39
ADR SEL HIGH: 0x49

Which one do have in use?

**Reply by sfromis** [reply]
Pick one of them, the driver supports all 3.

**Comment by Jason2866** [comment]
To be clear, there will be no software change made to output wrong values.

**Comment by lisergio** [comment]
Thanks to everyone
I will first try to put a filter in front of the sensor
